keyboards/handwired/replicazeron/replicazeron.c

```diff
@@ -62,14 +114,17 @@ bool oled_task_kb(void) {
     draw_oled(controller_state);
     return false;
 }
+
+oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+    return OLED_ROTATION_180;
+}
```

Comment:
this should be inside of a ifdef similar to RGBINDICATORS and optionally enabled from rules.mk